src/posts/keymaps/peterjc-hands_down_promethium.md

```diff
@@ -0,0 +1,25 @@
+---
+author: peterjc
+baseLayouts: ["Hands Down"]
+firmwares: ["Karabiner-Elements"]
+hasHomeRowMods: false
+hasLetterOnThumb: true
+hasRotaryEncoder: false
+isAutoShiftEnabled: false
+isComboEnabled: true
+isSplit: false
+isTapDanceEnabled: false
+keybindings: []
+keyboard: Japanese Apple KeyBoard
+keyCount: 35
```

Comment:
That's a core of 35 keys: 30 main keys, 2 left thumb keys, and 3 right thumb keys.

I am not counting Q & Z (available as combos and on the navigation & numbers layer), nor all the other keys still live on the keyboard like the number row (numberpad on layer), escape (combo), tab (combo), enter (combo), backspace (combo or left thumb), cursors (on layer).